<ui version="4.0" >
 <author>Andrey Rijov</author>
 <class>QEMU_Binary_Window</class>
 <widget class="QDialog" name="QEMU_Binary_Window" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Emulator Settings</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="images.qrc" >
    <normaloff>:/exec.png</normaloff>:/exec.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="0" column="0" colspan="3" >
    <layout class="QHBoxLayout" name="horizontalLayout_4" >
     <item>
      <widget class="QLabel" name="Label_Name" >
       <property name="text" >
        <string>Name (This be Identifier for it Emulator)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Edit_Name" />
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QWidget" native="1" name="widget" >
     <layout class="QHBoxLayout" name="horizontalLayout_3" >
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="Label_Find_Emulator" >
        <property name="text" >
         <string>Find Emulator:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="RB_QEMU" >
        <property name="text" >
         <string>QEMU</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="RB_KVM" >
        <property name="text" >
         <string>KVM</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>366</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3" >
    <layout class="QHBoxLayout" name="horizontalLayout_5" >
     <item>
      <widget class="QLabel" name="Label_Path_to_Dir" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Path to Emutators Binary Directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Edit_Path_to_Dir" />
     </item>
     <item>
      <widget class="QToolButton" name="TB_Browse" >
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="images.qrc" >
         <normaloff>:/open-folder.png</normaloff>:/open-folder.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="3" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="Label_Binary_Path" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Emulators Executables Files</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="3" >
    <widget class="QTableWidget" name="Table_QEMU_Systems" >
     <property name="editTriggers" >
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="verticalScrollMode" >
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode" >
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <column>
      <property name="text" >
       <string>QEMU/KVM System</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Binary File Name</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QWidget" native="1" name="Widget_QEMU" >
     <layout class="QGridLayout" name="gridLayout_2" >
      <property name="margin" >
       <number>0</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="Label_QEMU_Version" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>QEMU Version</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="QRadioButton" name="RB_Check_QEMU_Version" >
        <property name="text" >
         <string>Check and Set QEMU Version Always Start AQEMU</string>
        </property>
        <property name="checked" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QRadioButton" name="RB_Force_QEMU_Version" >
        <property name="text" >
         <string>Force QEMU Version</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QComboBox" name="CB_QEMU_Version" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex" >
         <number>0</number>
        </property>
        <property name="maxCount" >
         <number>256</number>
        </property>
        <item>
         <property name="text" >
          <string>QEMU 0.9.0</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>QEMU 0.9.1</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>QEMU 0.10.X</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>New</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="Line" name="line_5" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="2" >
    <widget class="QWidget" native="1" name="Widget_KVM" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="Label_KVM_Version" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>KVM Version</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="QRadioButton" name="RB_Check_KVM_Version" >
        <property name="text" >
         <string>Check and Set KVM Version Always Start AQEMU</string>
        </property>
        <property name="checked" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QRadioButton" name="RB_Force_KVM_Version" >
        <property name="text" >
         <string>Force KVM Version</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QComboBox" name="CB_KVM_Version" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex" >
         <number>0</number>
        </property>
        <property name="maxCount" >
         <number>256</number>
        </property>
        <item>
         <property name="text" >
          <string>KVM > 70</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>KVM > 80</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>New</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="3" >
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3" >
    <widget class="QCheckBox" name="CH_Check_Available_Audio_Cards" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Check Available Audio Cards Always Start AQEMU (This Incomplete!)</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3" >
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>298</width>
         <height>25</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Button_Find" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>&amp;Find Emulators</string>
       </property>
       <property name="icon" >
        <iconset resource="images.qrc" >
         <normaloff>:/find.png</normaloff>:/find.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_OK" >
       <property name="text" >
        <string>&amp;Apply Changes</string>
       </property>
       <property name="icon" >
        <iconset resource="images.qrc" >
         <normaloff>:/ok.png</normaloff>:/ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_Cancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="icon" >
        <iconset resource="images.qrc" >
         <normaloff>:/cancel.png</normaloff>:/cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>Edit_Name</tabstop>
  <tabstop>RB_QEMU</tabstop>
  <tabstop>RB_KVM</tabstop>
  <tabstop>Edit_Path_to_Dir</tabstop>
  <tabstop>TB_Browse</tabstop>
  <tabstop>Table_QEMU_Systems</tabstop>
  <tabstop>RB_Check_QEMU_Version</tabstop>
  <tabstop>RB_Force_QEMU_Version</tabstop>
  <tabstop>CB_QEMU_Version</tabstop>
  <tabstop>RB_Check_KVM_Version</tabstop>
  <tabstop>RB_Force_KVM_Version</tabstop>
  <tabstop>CB_KVM_Version</tabstop>
  <tabstop>CH_Check_Available_Audio_Cards</tabstop>
  <tabstop>Button_Find</tabstop>
  <tabstop>Button_OK</tabstop>
  <tabstop>Button_Cancel</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>Button_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>QEMU_Binary_Window</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>624</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RB_Force_QEMU_Version</sender>
   <signal>toggled(bool)</signal>
   <receiver>CB_QEMU_Version</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>132</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel" >
     <x>313</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RB_Force_KVM_Version</sender>
   <signal>toggled(bool)</signal>
   <receiver>CB_KVM_Version</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>459</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel" >
     <x>625</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RB_QEMU</sender>
   <signal>toggled(bool)</signal>
   <receiver>Widget_QEMU</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>173</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel" >
     <x>152</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RB_KVM</sender>
   <signal>toggled(bool)</signal>
   <receiver>Widget_KVM</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>237</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel" >
     <x>625</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
